core/options.cpp

```diff
@@ -155,7 +155,7 @@ bool Options::Parse(int argc, char *argv[])
 
 						auto fd_ctrl = cam->controls().find(&controls::FrameDurationLimits);
 						auto crop_ctrl = cam->properties().get(properties::ScalerCropMaximum);
-						double fps = 1e6 / fd_ctrl->second.min().get<int64_t>();
+						double fps = fd_ctrl==cam->controls().end() ? NAN : (1e6 / fd_ctrl->second.min().get<int64_t>());
 						std::cout << std::fixed << std::setprecision(2) << "["
```

Comment:
I think I would probably prefer 0 instead of NAN.  What do you think?

Reply:
Since 0 is a valid number, this could be interpreted as you can only take a single or no images. "NAN" signals for me that the framerate is not available or unknown. As a bonus, if someone decides to do math with it, the NAN will propagate and it will be easier visible that this operation should not be done.

Reply:
I don't think fps == 0.0 is possible for a valid configuration.  But I am ok with NAN either way.